pkg/ffmpeg/encoder_transcode.go

```diff
@@ -64,7 +64,7 @@ func (e *Encoder) Transcode(probeResult VideoFile, options TranscodeOptions) {
 		"-strict", "-2",
 		options.OutputPath,
 	}
-	_, _ = e.run(probeResult, args)
+	_, _, _ = e.run(probeResult, args)
```

Comment:
If the return values aren't used, just do `e.run(...)`. Multiple instances of this in this file.

Reply:
Fixed. Rebased and switched migration to 14 as well.